src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -577,6 +577,12 @@ public void BeginBuild(BuildParameters parameters)
                 // Log deferred messages and response files
                 LogDeferredMessages(loggingService, _deferredBuildMessages);
 
+                // Log if BuildCheck is enabled
+                if (_buildParameters.IsBuildCheckEnabled)
+                {
+                    loggingService.LogComment(buildEventContext: BuildEventContext.Invalid, MessageImportance.Normal, "BuildCheckEnabled");
```

Comment:
@JanKrivanek, @YuliiaKovalova we should probably do same for replaying binlog, right? We can do this in a different PR.

Reply:
Good point - it's good to have a feature parity.

It's not a high-pri though - as this was requested (for the security reasons) to inform about possible pluggability logic engagging as a part of the build process.